app-template/config-template.xml

```diff
@@ -42,44 +42,44 @@
+    <plugin name="cordova-plugin-x-socialsharing" spec="https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin.git#5.4.0" />
+    <plugin name="cordova-plugin-spinner-dialog" spec="1.3.1" />
+    <plugin name="cordova-plugin-dialogs" spec="https://github.com/apache/cordova-plugin-dialogs.git#2.0.1" />
     <plugin name="cordova-plugin-network-information" spec="https://github.com/apache/cordova-plugin-network-information.git" />
-    <plugin name="cordova-plugin-console" spec="https://github.com/apache/cordova-plugin-console.git" />
-    <plugin name="cordova-plugin-uniquedeviceid" spec="~1.3.2" />
-    <plugin name="cordova-plugin-file" spec="https://github.com/apache/cordova-plugin-file.git" />
-    <plugin name="cordova-plugin-touch-id" spec="https://github.com/EddyVerbruggen/cordova-plugin-touch-id.git" />
-    <plugin name="cordova-plugin-transport-security" spec="~0.1.2" />
-    <plugin name="cordova-ios-requires-fullscreen" spec="~0.0.2" />
-    <plugin name="cordova-plugin-android-fingerprint-auth" spec="~0.2.0" />
-    <plugin name="cordova-plugin-screen-orientation" spec="~1.4.2" />
-    <plugin name="ionic-plugin-keyboard" spec="~2.2.1" />
-    <plugin name="cordova-plugin-whitelist" spec="~1.3.0" />
-    <plugin name="cordova-plugin-ionic-webview" spec="~3.0.0" />
+    <plugin name="cordova-plugin-console" spec="https://github.com/apache/cordova-plugin-console.git#1.1.0" />
+    <plugin name="cordova-plugin-uniquedeviceid" spec="1.3.2" />
+    <plugin name="cordova-plugin-file" spec="https://github.com/apache/cordova-plugin-file.git#6.0.1" />
+    <plugin name="cordova-plugin-touch-id" spec="https://github.com/EddyVerbruggen/cordova-plugin-touch-id.git#3.3.1" />
+    <plugin name="cordova-plugin-transport-security" spec="0.1.2" />
+    <plugin name="cordova-ios-requires-fullscreen" spec="0.0.2" />
+    <plugin name="cordova-plugin-android-fingerprint-auth" spec="0.2.0" />
+    <plugin name="cordova-plugin-screen-orientation" spec="1.4.2" />
+    <plugin name="ionic-plugin-keyboard" spec="2.2.1" />
+    <plugin name="cordova-plugin-whitelist" spec="1.3.3" />
+    <plugin name="cordova-plugin-ionic-webview" spec="2.2.3" />
     <plugin name="cordova-plugin-qrscanner-no-android" spec="https://github.com/Bitcoin-com/cordova-plugin-qrscanner-no-android.git" commit="cbb6910d245a60899d33d933c9fa369c960e90f7" />
-    <plugin name="cordova-plugin-camera-preview" spec="~0.10.0" />
-    <plugin name="cordova-plugin-customurlscheme" spec="https://github.com/cmgustavo/Custom-URL-scheme.git">
+    <plugin name="cordova-plugin-camera-preview" spec="0.10.0" />
+    <plugin name="cordova-plugin-customurlscheme" spec="https://github.com/cmgustavo/Custom-URL-scheme.git#4.1.5">
       <variable name="URL_SCHEME" value="bitcoin" />
-      <variable name="SECOND_URL_SCHEME" value="*APPURI*" />
+      <variable name="SECOND_URL_SCHEME" value="bitcoincom" />
```

Comment:
I don't think it is an expected behavior.

Reply:
I think it might be a mistake, there is no reason to change this